django/publicmapping/redistricting/urls.py

```diff
@@ -22,53 +22,55 @@
 """
 
-from django.conf.urls.defaults import *
+from django.conf.urls import url, include
 from django.conf import settings
 
-urlpatterns = patterns('publicmapping.redistricting.views',
-    (r'^$', 'viewplan', { 'planid': 0 }),
-    (r'^plan/create/$', 'createplan'),
-    (r'^plan/upload/$', 'uploadfile'),
-    (r'^plan/(?P<planid>\d*)/view/$', 'viewplan'),
-    (r'^plan/(?P<planid>\d*)/edit/$', 'editplan'),
-    (r'^plan/(?P<planid>\d*)/print/$', 'printplan'),
-    (r'^plan/(?P<planid>\d*)/getreport/$', 'getreport'),
-    (r'^plan/(?P<planid>\d*)/getcalculatorreport/$', 'getcalculatorreport'),
-    (r'^plan/(?P<planid>\d*)/attributes/$', 'editplanattributes'),
-    (r'^plan/(?P<planid>\d*)/delete/$', 'deleteplan'),
-    (r'^plan/(?P<planid>\d*)/copy/$', 'copyplan'),
-    (r'^plan/(?P<planid>\d*)/unload/$', 'unloadplan'),
-    (r'^plan/(?P<planid>\d*)/score/$', 'scoreplan'),
-    (r'^plan/(?P<planid>\d*)/reaggregate/$', 'reaggregateplan'),
-    (r'^plan/(?P<planid>\d*)/district/(?P<districtid>\d*)/add/', 'addtodistrict'),
-    (r'^plan/(?P<planid>\d*)/district/(?P<district_id>\d*)/lock/', 'setdistrictlock'),
-    (r'^plan/(?P<planid>\d*)/district/(?P<district_id>\d*)/info/$', 'district_info'),
-    (r'^plan/(?P<planid>\d*)/demographics/$', 'get_statistics'),
-    (r'^plan/(?P<planid>\d*)/districts/$', 'getdistricts'), 
-    (r'^plan/(?P<planid>\d*)/district/new/$', 'newdistrict'),
-    (r'^plan/(?P<planid>\d*)/shareddistricts/$', 'get_shared_districts'), 
-    (r'^plan/(?P<planid>\d*)/pastedistricts/$', 'add_districts_to_plan'),
-    (r'^plan/(?P<planid>\d*)/districtmembers/$', 'assign_district_members'),
-    (r'^plan/(?P<planid>\d*)/combinedistricts/$', 'combine_districts'),
-    (r'^plan/(?P<planid>\d*)/fixunassigned/$', 'fix_unassigned'),
-    (r'^plan/(?P<planid>\d*)/district/versioned/$', 'simple_district_versioned'),
-    (r'^plan/(?P<planid>\d*)/unlockedgeometries/$', 'get_unlocked_simple_geometries'),
-    (r'^plan/(?P<planid>\d*)/districtfile/$', 'getdistrictfile'),
-    (r'^plan/(?P<planid>\d*)/districtindexfilesend/$', 'emaildistrictindexfile'),
-    (r'^plan/(?P<planid>\d*)/districtfilestatus/$', 'getdistrictfilestatus'),
-    (r'^plan/(?P<planid>\d*)/statisticssets/$', 'statistics_sets'),
-    (r'^plan/(?P<planid>\d*)/splits/(?P<othertype>\w+)/(?P<otherid>\d*)/$', 'get_splits'),
-    (r'^plan/(?P<planid>\d*)/splitsreport/$', 'get_splits_report'),
-    (r'^getplans/$', 'getplans'),
-    (r'^getleaderboard/$', 'getleaderboard'),
-    (r'^getleaderboardcsv/$', 'getleaderboardcsv'),
-    (r'^health/$', 'get_health'),
-    (r'^processingstatus/$', 'get_processing_status'),
-    (r'^feed/plans/$', 'plan_feed'),
-    (r'^feed/shared/$', 'share_feed'),
-    (r'^i18n/', include('django.conf.urls.i18n')),
-)
+from .  import views as redistricting_views
+
+urlpatterns = [
+    url(r'^plan/$', redistricting_views.viewplan, { 'planid': 0 }),
+    url(r'^plan/create/$', redistricting_views.createplan),
```

Comment:
The removal of the `name` parameters here may be the source of the confusion with `reverse()` if we were relying on calling these views by name. In other words, this line may need to become `url(r'^plan/create/$', redistricting_views.createplan, name='createplan')`.

Reply:
I tried adding names to all of the url views including the login view with no change, though I may not have understood correctly what the names should have been